schemas/imaging/imaging_processing.json

```diff
@@ -0,0 +1,423 @@
+                                    "type": "string",
+                                    "title": "Volume input plugin"
+                                }
+                            },
+                            "required": [
+                                "channel_excitation",
+                                "channel_emission",
+                                "reference_1",
+                                "voxel_size_1",
+                                "reference_2",
+                                "voxel_size_2",
+                                "reference_3",
+                                "voxel_size_3",
+                                "stitching_channel",
+                                "volume_input_plugin"
+                            ]
+                        },
+                        {
+                            "title": "align",
+                            "type": "object",
+                            "properties": {
+                                "channel_excitation": {
+                                    "type": "integer",
+                                    "title": "Channel excitation (nm)",
+                                    "units": "nm"
+                                },
+                                "channel_emission": {
+                                    "type": "integer",
+                                    "title": "Channel emission (nm)",
+                                    "units": "nm"
+                                },
+                                "parallel_params": {
+                                    "type": "array",
+                                    "items": {"$ref": "#/$defs/parallel"},
+                                    "uniqueItems": true,
+                                    "title": "Parallel parameters"
+                                },
+                                "subvoldim": {
+                                    "type": "integer",
+                                    "title": "Subvolume dimension",
+                                    "units": "slices"
+                                }
+                            },
+                            "required": [
+                                "excitation",
+                                "emission",
+                                "parallel_params",
+                                "subvoldim"
+                             ]
+                        },
+                        {
+                            "title": "merge_volume",
+                            "type": "object",
+                            "properties": {
+                                "parallel_params": {
+                                    "type": "array",
+                                    "items": {"$ref": "#/$defs/parallel"},
+                                    "uniqueItems": true,
+                                    "title": "Parallel parameters"
+                                },
+                                "s": {
```

Comment:
yikes

Reply:
yes, Camillo doesn't know what they are ... so right now we're stuck with them